lib/omnibus/software.rb

```diff
@@ -696,6 +750,11 @@ def overridden?
       @overrides.key?(:version) && (@overrides[:version] != @version)
     end
 
+    def build_dep
+      @build_dep
+    end
+    expose :build_dep
```

Comment:
We should name this `build_dependency` so it matches with the naming idioms of `dependency`.